keyboards/nifty_numpad/config.h

```diff
@@ -0,0 +1,18 @@
+// Copyright 2023 Isaac Rex (@Acliad)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#define EECONFIG_KB_DATA_SIZE 11
+
+// RGB Matrix Parameters
+#define RGB_MATRIX_DEFAULT_MODE RGB_MATRIX_SOLID_COLOR // Sets the default mode, if none has been set
+#define RGB_MATRIX_DEFAULT_SPD 100 // Sets the default animation speed, if none has been set
+#define RGB_MATRIX_MAXIMUM_BRIGHTNESS 255
+
+#define RGB_MATRIX_DEFAULT_HUE 127 // Sets the default hue value, if none has been set
+#define RGB_MATRIX_DEFAULT_SAT 255 // Sets the default saturation value, if none has been set
+#define RGB_MATRIX_DEFAULT_VAL RGB_MATRIX_MAXIMUM_BRIGHTNESS // Sets the default brightness value, if none has been set
```

Comment:
These can all be migrated to info.json.  `qmk migrate` or `qmk info -kb (name) -f json` can assist with that.